Arduino/micro-ros_runservo_service/micro-ros_runservo_service.ino

```diff
@@ -0,0 +1,107 @@
+#include <micro_ros_arduino.h>
+#include <servo_interfaces/srv/run_servo.h>
+#include <stdio.h>
+#include <rcl/error_handling.h>
+#include <rclc/rclc.h>
+#include <rclc/executor.h>
+#include <micro_ros_utilities/string_utilities.h>
+#include <Servo.h>
+
+Servo servo1, servo2, servo3, servo4;
+
+
+rcl_node_t node;
+rclc_support_t support;
+rcl_allocator_t allocator;
+rclc_executor_t executor;
+
+rcl_service_t service;
+rcl_wait_set_t wait_set;
+
+servo_interfaces__srv__RunServo_Response res;
+servo_interfaces__srv__RunServo_Request req;
+
+#define RCCHECK(fn) { rcl_ret_t temp_rc = fn; if((temp_rc != RCL_RET_OK)){while(1){};}}
+#define RCSOFTCHECK(fn) { rcl_ret_t temp_rc = fn; if((temp_rc != RCL_RET_OK)){}}
+
+void service_callback(const void * req, void * res){
+  servo_interfaces__srv__RunServo_Request * req_in = (servo_interfaces__srv__RunServo_Request *) req;
+  servo_interfaces__srv__RunServo_Response * res_in = (servo_interfaces__srv__RunServo_Response *) res;
+
+  //printf("Service request value: %d + %d.\n", (int) req_in->a, (int) req_in->b);
+
+  //res_in->sum = req_in->a + req_in->b;
+
```

Comment:
why is this  a switch statement and not a function? I don't see any difference other than servo1-4

Reply:
We can make it into a function but different motors need different microseconds to do a full 360 degree turn.